### ISSUE TITLE
Contact us link on mobile nav doesn't navigate to contact us section when coming from blog page

### ISSUE THREAD

**Issue by samsmith1983** [redbadger/website-honestly#295]
Since #276 the 'contact us' link in the mobile nav menu should take you to the contact slice on the home page, rather than just to the top of the home page.

This is working everywhere apart from when coming from the blog page.

Repro:

- Go to blog page
- If not on mobile, resize browser to mobile size
- Open mobile nav menu
- Click 'contact us'

You are just taken to the top of the homepage

**Comment by grahammendick** [comment]
Talked with @asavin and we decided we'd change the menu in the template on squarespace to point at `https://red-badger.com/#contactUs`. But I noticed that the squarespace template doesn't event [point at the home page](https://github.com/redbadger/blog-squarespace-template/blob/master/site.region#L77). It points at a non-existent page and then there is a [proxy rule in place](https://github.com/redbadger/website-next-proxy/blob/master/nginx/proxy.template#L229) that redirects to the home page.

Should we fix this by changing the squarespace template and deleting the proxy rule? Or should we just change the proxy rule to rewrite to the correct location?
